content/en/blog/2024/otel-collector-survey/index.md

```diff
@@ -0,0 +1,93 @@
+
+## Key Takeaways
+- Companies typically have medium to large collector deployments:
+  - \>5 Collectors: 125/186
+  - \>10 Collectors: 100/186
+- Building custom binaries/distributions of the Collector is more popular than expected (61/186), with most people using the [OTel Collector Builder](https://github.com/open-telemetry/opentelemetry-collector-builder) to do so (49/61)
+- The vast majority deploy the Collector on Kubernetes (150/186)
+- There's more desire for stability (59), self-observability (53), and config management (59), than for new components (14)
+
+## Detailed Insights
+
+### Deployment Scale and Environment
+Our findings indicate robust use of the OTel Collector at scale, with 53.8%(100/186) of respondents deploying more than 10 collectors, and  22%(41/186) of respondents running between 2 and 5 collectors
+
+![Chart showing how many otel collectors people run in their organization](deployment-scale.png)
+
+Kubernetes is the leading platform for Collector deployment (80.6%), followed by virtual machines (33.3%) and bare metal (10.8%).
+
+![Chart showing where people deploy their otel collectors](deployment-environment.png)
+
+### Usage Scenarios 
+The OTel Collector is primarily used as a gateway (64.5%), demonstrating its pivotal role in aggregating telemetry data from various sources. Daemonsets (51.6%) and sidecars (23.7%) are also popular deployment models, showcasing the flexibility of the OTel Collector in different operational contexts.
+
+![Chart showing the usage scenarios of the otel collector](usage-scenarios.png)
+
+### Customization and Configuration
+A surprising number of people build their own distributions of the Collector (61/186), indicating that providing a composable Collector is important for the community. Most people who do build their own Collector distros use the [OTel Collector Builder (OCB)](https://github.com/open-telemetry/opentelemetry-collector-builder) (49/61). Of the 49 respondents who leverage the OCB, most were able to figure it out, with only 2 respondents stating that it’s too hard to use.
+
+![Chart showing how easy people find it to use the otel collector builder](ocb-usage.png)
+
+### Monitoring and Observability
+When it comes to monitoring the Collectors, a vast majority of respondents rely on Collector metrics and logs (70%), and only a few don't monitor their Collector(s) at all (16%). When we peered deeper into the data, we found that of the 125 respondents with more than 5 Collectors, only 15 are not monitoring their Collectors, and of the 100 respondents with more than 10 Collectors, only 9 are not monitoring their Collectors. This seems to indicate that users take monitoring their Collectors seriously once they reach a certain maturity with their Collector deployments.
+
+![Chart showing how people monitor their otel collector](monitoring.png)
```

Comment:
This graph is too crowded, with a lot of 1-answers. How about grouping all those 1-answers into an "Others" item? That'd be 13 lines, which account for less than 10% of the usage, but a good chunk of the image.

Reply:
Done